File: app/src/main/java/com/rdev/tryp/model/favorite_address/FavoriteAddress.java
package com.rdev.tryp.model.favorite_address;

import com.google.gson.annotations.SerializedName;

public class FavoriteAddress {

        @SerializedName("data")
        private Data data;

        public void setData(Data data){
            this.data = data;
        }

        public Data getData(){
            return data;
        }

        @Override
        public String toString(){
            return
                    "FavoriteAddress{" +
                            "data = '" + data + '\'' +
                            "}";
        }
}

File: app/src/main/java/com/rdev/tryp/model/favorite_address/FavoriteAddress.kt
package com.rdev.tryp.model.favorite_address

import com.google.gson.annotations.SerializedName


class FavoriteAddress {

    @SerializedName("data")
    var data: Data? = null

    override fun toString(): String {
        return "FavoriteAddress{" +
                "data = '" + data + '\''.toString() +
                "}"
    }

}